cmd/retrieval-client/main.go

```diff
@@ -95,14 +128,11 @@ func run(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-
-	log.Info("submit query ", params)
-
 	for {
 		select {
 		case resp := <-h.respCh:
 			log.Info("got response! ", resp)
-		case <-time.After(responseTimeout):
+		case <-time.After(time.Duration(timeout)):
```

Comment:
why the conversion?

Reply:
also, it seems to timeout instantly now, since `timeout` is now `60` and `time.Duration` is in nanoseconds, could you change `defaultResponseTimeout = time.Minute.Seconds()`  back to `time.Minute`?

Reply:
I think the CLI should ask for seconds, which is why I converted it here. I think I've fixed the issue with this being parsed, lmk what you think